main/SetupNG.h

```diff
@@ -615,7 +615,9 @@ extern SetupNG<int> 		wk_label_minus_1;
 extern SetupNG<int> 		wk_label_minus_2;
 extern SetupNG<int> 		wk_label_minus_3;
 extern SetupNG<float>       flap_takeoff;
-extern SetupNG<int> 		audio_disable;
+extern SetupNG<int> 		audio_mute_menu;  // replaced audio_disable (same key)
```

Comment:
Ahh i see, same key, so this should work with older sw loads for in service upgrade.